qhana_plugin_registry/api/models/templates.py

```diff
@@ -54,7 +54,7 @@ class TemplateTabSchema(ApiObjectSchema):
     description = ma.fields.String(required=True, allow_none=False)
     location = ma.fields.String(required=True, allow_none=False, validate=Length(max=255))
     sort_key = ma.fields.Integer(required=True, allow_none=False, default=0)
-    plugin_filter = ma.fields.String(required=True, allow_none=False)
+    filter_string = ma.fields.String(required=True, allow_none=False)
```

Comment:
You could use a custom field to handle the validation like the Version field in this example: <https://marshmallow.readthedocs.io/en/stable/examples.html#validating-package-json>

Reply:
I ended up using the `@validates` decorator to validate the `filter_string` field (3622b0496d1bb855c5854211397ddddcd0e4741d).